### PR TITLE
[W03] Add Test Correct Size function of Dynamic Stack

### PR DESCRIPTION
**Pull request by faid0**
Add Test Correct Size function of Dynamic Stack

#### General

- [x] I confirm that my tests, if any, are correct
    - [x] Optional: I pass **all** artemis tests.
    - [x] Optional: I pass **all** tests, that are added in this PR.
    - [ ] Optional: I calculated manually the correct solutions on paper.
- [x] I chose a title conforming to the [naming conventions for pull requests](https://github.com/JohannesStoehr/gad23-tests/blob/main/CONTRIBUTING.md).
- [x] The newly added tests do not reveal the solutions dynamically calculating the solutions as expected in the exercise.
- [ ] Other students approved this Pull Request.

### CODE REVIEW

File: w03/test/gad/dynamicarray/DynamicStackCorrectSizeTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package gad.dynamicStack;
+
+import gad.dynamicarray.DynamicStack;
+import gad.dynamicarray.StudentResult;
```
Comment (felixhauptmann) [inline w03/test/gad/dynamicarray/DynamicStackCorrectSizeTest.java:4]:
```suggestion
```

remove unused imports

**Review by felixhauptmann — APPROVED** [review]
LGTM!

File: w03/test/gad/dynamicarray/DynamicStackCorrectSizeTest.java (hunk 1 above)
Comment (felixhauptmann) [reply w03/test/gad/dynamicarray/DynamicStackCorrectSizeTest.java:4]: autoformatting would be great too... :)

**Review by nilsreichardt — APPROVED** [review]
LGTM

**Comment by nilsreichardt** [comment]
cc: @JohannesStoehr

**Review by Anatoly03 — APPROVED** [review]

**Pull request merged**